misc/atrium-specs/src/commonMain/kotlin/ch/tutteli/atrium/specs/integration/AbstractCollectionExpectationsTest.kt

```diff
@@ -0,0 +1,74 @@
+package ch.tutteli.atrium.specs.integration
+
+import ch.tutteli.atrium.api.fluent.en_GB.*
+import ch.tutteli.atrium.creating.Expect
+import ch.tutteli.atrium.specs.*
+import ch.tutteli.atrium.specs.integration.utils.ExpectationCreatorTestData
+import ch.tutteli.atrium.specs.integration.utils.SubjectLessTestData
+import ch.tutteli.atrium.testfactories.TestFactory
+import ch.tutteli.atrium.translations.DescriptionCollectionExpectation
+
+@Suppress("FunctionName")
+abstract class AbstractCollectionExpectationsTest(
+    private val isEmptySpec: Fun0<Collection<Int>>,
```

Comment:
looks like we still used an old naming here, could you please replace `is` with toBe

```suggestion
    private val toBeEmptySpec: Fun0<Collection<Int>>,
```

Reply:
@robstoll - Done.